file/file.go

```diff
@@ -0,0 +1,70 @@
+// Package file - Handle file operations
```

Comment:
comment format is similar to public package members. (I'm actually not sure, you should check in in one of the effective go or code review comments docs I sent you).
```
// <pkg_name> <the rest of the sentence>

// file handle file operations.
```